src/widget.cpp

```diff
@@ -2957,192 +2957,189 @@ bool NWidgetLeaf::ButtonHit(const Point &pt)
 /* == Conversion code from NWidgetPart array to NWidgetBase* tree == */
 
 /**
- * Construct a single nested widget in \a *dest from its parts.
- *
- * Construct a NWidgetBase object from a #NWidget function, and apply all
- * settings that follow it, until encountering a #EndContainer, another
- * #NWidget, or the end of the parts array.
- *
- * @param nwid_begin Iterator to beginning of nested widget parts.
- * @param nwid_end Iterator to ending of nested widget parts.
- * @param dest  Address of pointer to use for returning the composed widget.
- * @param fill_dest Fill the composed widget with child widgets.
- * @return Iterator to remaining nested widget parts.
+ * Test if (an NWidgetPart) WidgetType is an attribute widget part type.
```

Comment:
```suggestion
 * Test if (a NWidgetPart) WidgetType is an attribute widget part type.
```

Reply:
Hmm, so I pronounce this as "en-widget-part", which therefore requires "an" in English.

Reply:
Fine by me